st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
+                "type": "array",
+                "items": {"type": "string"},
+                "description": "Parameter format hidden from public list."
+            },
+            "ack": {
+                "type": "object",
+                "properties": {
+                    "enabled": {"type": "boolean"},
+                    "format": {"type": "string"}
+                },
+                "description": "Acknowledgement message format."
+            },
+            "result": {
+                "type": "object",
+                "properties": {
+                    "enabled": {"type": "boolean"},
+                    "format": {"type": "string"}
```

Comment:
```
format: "your execution id is {{ execution.id }} and the result is {{ execution.result }}"
```
Something like that. @enykeev is working on the back-end changes for that, and `ack` parameters (both `enable` and `format`) are (mostly) implemented in my recent `hubot-stackstorm` PR.